packages/sdk-core/tsconfig.module.json

```diff
@@ -0,0 +1,14 @@
+{
+    "extends": "./tsconfig",
+    "compilerOptions": {
+        "target": "ES2020",
+        "outDir": "dist/module",
+        "module": "esnext",
+        "declaration": true,
+        "declarationMap": true,
+        "sourceMap": true
```

Comment:
Generating source maps and declaration maps now.

Reply:
🔥